### ISSUE TITLE
Erro no prompt de comando

### ISSUE THREAD

**Issue by brunochagas2010** [RodrigoMenegat/controle-seus-tweets#2]
Esse erro apareceu pra mim no windows 7 quando executei o comando pip install -r requirements.txt
Não entendi o que deu errado.

![erro](https://user-images.githubusercontent.com/57696126/68802360-8c4ecb80-063c-11ea-8d22-ca321eac1bbf.png)

**Comment by RodrigoMenegat** [comment]
Oi! Pelo visto, o que aconteceu é que sua internet bloqueou a conexão com o site onde ficam os pacotes que precisamos instalar. Você está usando ela sob algum proxy, como na rede do trabalho ou algo do gênero? Pode ser que essa seja a causa.

**Comment by brunochagas2010** [comment]
Certo. Estou fazendo de uma rede do trabalho mesmo. Mais tarde vou testar em casa! Valeu pelo retorno!

**Comment by RodrigoMenegat** [comment]
Caso ainda tenha problemas, só falar por aqui :)
